tests/std/tests/P0896R4_ranges_alg_fill_n/test.cpp

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <algorithm>
+#include <cassert>
+#include <concepts>
+#include <range_algorithm_support.hpp>
+#include <ranges>
+#include <utility>
+
+constexpr void smoke_test() {
+    using ranges::fill_n, ranges::iterator_t;
+    using std::same_as;
+
+    int output1[] = {13, 42, 1367};

```

Comment:
Should this just be `output` since there is no second array?

Reply:
Similarly, should we unbrace the following code block since it's the only block in this function and there's nothing to isolate it from?